### PR TITLE
Update verbiage to proper tense for feature deprecations

### PR DESCRIPTION
**Pull request by kkirsche**
X-Ref: https://github.com/twbs/bootstrap/commit/fce30b57e2fbe750cdebf1794d3917f3176d06ce

### CODE REVIEW

**Comment by cvrebert** [comment]
Seems like these should say "has been removed".

**Comment by kkirsche** [comment]
Updated per your comment @cvrebert. Sorry for that.

File: less/mixins/vendor-prefixes.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Vendor Prefixes
 //
 // All vendor mixins are deprecated as of v3.2.0 due to the introduction of
-// Autoprefixer in our Gruntfile. They will be removed in v4.
+// Autoprefixer in our Gruntfile. They has been removed in v4.
```
Comment (cvrebert) [inline less/mixins/vendor-prefixes.less]: "**have** been"

**Comment by kkirsche** [comment]
Updated per your comment @cvrebert. I apologize for the inconvenience.

**Comment by cvrebert** [comment]
No worries.

**Comment by cvrebert** [comment]
Thanks

**Pull request merged**